docs/docs/20-usage/90-advanced-usage.md

```diff
@@ -127,3 +127,44 @@ WOODPECKER_ENVIRONMENT=first_var:value1,second_var:value2
 ```
 
 Note that this tightly couples the server and app configurations (where the app is a completely separate application). But this is a good option for truly global variables which should apply to all steps in all pipelines for all apps.
+
+## Docker in docker (dind) setup
+
+:::warning
+This set up will only work on trusted repositories and for security reasons should only be used in private environments. See [project settings](./75-project-settings.md#trusted) to enable trusted mode.
+:::
+
+The snippet below shows how a step can communicate with the docker daemon via a `docker:dind` service.
+
+:::note
+If your aim ist to build/publish OCI images, consider using the [Docker Buildx Plugin](https://woodpecker-ci.org/plugins/Docker%20Buildx) instead.
+:::
+
+The most important bit ist to allow the dind service to write/read certificates on a mounted volume that is also shared with the step that will contact the daemon (it needs the service's CA to do so successfully).
```

Comment:
```suggestion
The most important bit is to allow the `dind` service to write/read certificates on a mounted volume that is also shared with the step that will contact the daemon (it needs the service's CA to do so successfully).
```

TLS is enforced since docker v27 AFAIR. Maybe a short link to that can help to explain this requirement a bit better with less complicated/technical wording?

Reply:
Fair point! i added a note about it in [212f29e](https://github.com/woodpecker-ci/woodpecker/pull/4620/commits/212f29e2045c04ded666be1497d0a3098b344b9a)